dcos_installer/test_backend.py

```diff
@@ -67,6 +67,38 @@ def test_set_superuser_password(tmpdir):
         assert passlib.hash.sha512_crypt.verify('foo', config['superuser_password_hash'])
 
 
+def test_generate_node_upgrade_script(tmpdir):
```

Comment:
Testing that a missing version number causes the script to exit with an error status would be useful.